libc/CMakeLists.txt

```diff
@@ -725,9 +750,9 @@ target_link_libraries(oelibc
   PRIVATE oelibasm)
 
 target_include_directories(oelibc PRIVATE
-  internal
```

Comment:
I don't remember this deletion from the original branch. What necessitated this?

Reply:
Turns out there is no such folder as libc/internal. 
We were just adding an invalid folder to the include path.